zk_toolbox/crates/zk_inception/src/commands/chain/args/create.rs

```diff
@@ -73,32 +75,48 @@ impl ChainCreateArgs {
             .unwrap_or_else(|| Prompt::new(MSG_CHAIN_NAME_PROMPT).ask());
         chain_name = slugify!(&chain_name, separator = "_");
 
-        let chain_id = self.chain_id.unwrap_or_else(|| {
-            Prompt::new(MSG_CHAIN_ID_PROMPT)
-                .default(&(L2_CHAIN_ID + number_of_chains).to_string())
-                .ask()
+        let chain_id = self
+            .chain_id
+            .map(|v| {
+                (v == 0)
+                    .then(|| L2_CHAIN_ID + number_of_chains)
+                    .unwrap_or(v)
+            })
```

Comment:
i don't really like this approach with predefined values, should we either, make it a string and check that it's either numeric value or `default` or add an enum, which probably will do the same automatically

Reply:
Addressed in #2579